test/development/acceptance-app/rsc-build-errors.test.ts

```diff
@@ -350,6 +350,34 @@ describe('Error overlay - RSC build errors', () => {
     await cleanup()
   })
 
+  it('should display error message when module resolve error occurred', async () => {
+    const { session, cleanup } = await sandbox(
+      next,
+      new Map([
+        [
+          'app/unresolve/page.js',
+          `
+          import Mod from 'non-existing-module'
+
+          export default function Page() {
+            return <p>page</p>
+          }
+        `,
+        ],
+      ]),
+      '/unresolve'
+    )
+
+    await session.assertHasRedbox()
+    if (!isTurbopack) {
```

Comment:
What happens in turbopack?

Reply:
turbopack has the filepath right before error message so for this case it's displaying the file path right now. I leave a message that we'd like to fix it